src/electron/flux/store/android-setup-store.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BaseStoreImpl } from 'background/stores/base-store-impl';
+import { StoreNames } from 'common/stores/store-names';
+import { AndroidSetupActions } from '../action/android-setup-actions';
+import { AndroidSetupStoreData } from '../types/android-setup-store-data';
+
+export class AndroidSetupStore extends BaseStoreImpl<AndroidSetupStoreData> {
+    constructor(private readonly androidSetupActions: AndroidSetupActions) {
+        super(StoreNames.AndroidSetupStore);
+    }
+
+    public getDefaultState(): AndroidSetupStoreData {
+        return { currentStepId: 'detect-adb' };
+    }
+
+    private onCancel = () => {
+        this.emitChanged();
```

Comment:
This isn't doing anything until the state machine gets in. I'm open to either setting the current step to something (like the choose-device screen, which is eventual desired behavior) or removing if folks have strong opinions.

Reply:
I think this is fine for placeholder behavior